pkg/aggregator/walker.go

```diff
@@ -24,6 +24,7 @@ import (
 
 const (
 	definitionPrefix = "#/definitions/"
+	parameterPrefix  = "#/parameters/"
```

Comment:
I didn't know this was a thing: https://swagger.io/specification/v2/#parameters-definitions-object :-)

Reply:
> OpenAPI - limitless expansiveness... A vast realm of expressiveness, its boundaries extending beyond the conventional. We're venturing into a new era of API specifications, to explore strange new ways of defining APIs, to seek out and implement new protocols and new schemas, to boldly express what no one has expressed before..."